server/controllers/feed.controller.js

```diff
@@ -6,7 +6,11 @@ import Feed from '../models/feed.model';
 import RelatedLink from '../models/relatedLink.model'
 
 function list(req, res, next) {
-  Feed.findOne({user: req.user._id})
+
+  // If user is logged out, use Jeff's feed TODO cleaner solution
+  let userId = req.user ? req.user._id : '597a06d7f0dc67003db0c4c0'
```

Comment:
Is this a user in production or staging?

Reply:
It's one I copied from prod to local